WordPress/Classes/Models/UserProfile.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+open class UserProfile {
```

Comment:
True but I wouldn't be a fan of an initializer that required 10 arguments to use.  Nate also suggested a struct vs a class, so let's go with that change.